_sass/_theme.scss

```diff
@@ -104,14 +104,33 @@ body {
 .footer {
   background: $headingColour;
   color: $captionColour;
+  position: fixed;
```

Comment:
Why? I don't know if I like a fixed footer.

Reply:
It was requested in https://matrix.to/#/#aula:pikaviestin.fi to promote Kapsi and a PR was welcomed. I'm largely indifferent.